packages/patternfly-react/src/components/MessageDialog/MessageDialog.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { noop } from '../../common/helpers';
+import { Modal, Button } from '../../index';
+
+const MessageDialog = ({
+  show,
+  onHide,
+  primaryAction,
+  secondaryAction,
+  title,
+  icon,
+  primaryContent,
+  secondaryContent,
+  primaryActionButtonBsStyle,
+  secondaryActionButtonBsStyle,
+  primaryActionButtonContent,
+  secondaryActionButtonContent,
+  className,
+  footer,
+  ...props
+}) => (
+  <Modal
+    className={classNames('message-dialog-pf', className)}
+    show={show}
+    onHide={onHide}
+    enforceFocus
```

Comment:
I think this should be a prop and defaulted to true so that applications can override. Especially given the warning in the react bootstrap docs:

> When true The modal will prevent focus from leaving the Modal while open. Consider leaving the default value here, as it is necessary to make the Modal work well with assistive technologies, such as screen readers.